src/e-build.js

```diff
@@ -92,6 +92,7 @@ program
   .option('--gen', 'Force a re-run of `gn gen` before building', false)
   .option('-t|--target [target]', 'Forces a specific ninja target')
   .option('--no-goma', 'Build without goma', false)
+  .option('--no-load-xcode', 'Do not load Xcode prior to building', false)
```

Comment:
@codebytere is this option used?  And if so how?

Reply:
@jkleinsc at the moment `e-build` automatically runs `e-load-xcode` with no arguments which would override the version that had just prior been set